documentation/source/quickstart.rst

```diff
@@ -1,123 +1,16 @@
+.. _quickstart:
```

Comment:
Should be no need for this

Reply:
Strictly speaking yes, but I wanted these anchors to be more resilient against file renamings, something that may happen as we restructure more.